ga4gh/datamodel/__init__.py

```diff
@@ -324,25 +324,6 @@ class PhenotypeAssociationCompoundId(CompoundId):
         ('phenotypeAssociationId', 2)]
 
 
-class PhenotypeAssociationSetCompoundId(CompoundId):
```

Comment:
@david4096: thanks for the +1.  Regarding additional entries for other identifiers, I considered it for Phenotype.  After looking at the use cases, I didn't see the need.  The most likely future use that might require it is GET ../genotype/[id]. I'd recommend deferring that until the need materializes.